ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -97,6 +97,23 @@ public class BottomSheetController: UIViewController {
         }
     }
 
+    /// Indicates if the sheet height is flexible.
+    ///
+    /// When set to `false`, the sheet height is static and always corresponds to the height of the maximum expansion state.
+    /// Interacting with the sheet will only vertically translate it, moving it partially on/off-screen.
+    ///
+    /// When set to `true`, moving the sheet beyond the `.collapsed` state will resize it.
+    ///
+    /// Use flexible height if you have views attached to the bottom of `expandedContentView` that should always be visible.
+    @objc open var isFlexibleHeight: Bool = false {
+        didSet {
+            guard isViewLoaded else {
+                return
+            }
+            view.setNeedsLayout()
```

Comment:
Why is this needed? I would have expected toggling this doesn't change anything about its current position and only affects what happens when interacted with? It sounds like this will also cause a one-time unanimated snap to allowed conditions that may be worth documenting?

Reply:
The sheet frame needs to be recalculated, because everywhere except fully expanded state the frame will be different in flexible mode. For example if we're in collapsed state, setting this to true will resize the sheet so that its bottom is aligned with the bottom of the screen. Its offset wouldn't change in this case.

This won't cause a snap if the sheet is being interacted with because it'll be in the `.transitioning` state. In that case the interaction will automatically switch to calculating the frame using the new flexible / rigid mode.